e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/deal-cancellation-enabled/dashboard-deals-filter-by-cancellation-status.spec.js

```diff
@@ -0,0 +1,93 @@
+const { TFM_DEAL_STAGE } = require('@ukef/dtfs2-common');
+const relative = require('../../../../relativeURL');
+const MOCK_USERS = require('../../../../../../../e2e-fixtures');
+const { dashboardDeals } = require('../../../../pages');
+const { dashboardFilters } = require('../../../../partials');
+const { BSS_DEAL_READY_FOR_CHECK, GEF_DEAL_DRAFT, BSS_DEAL_CANCELLED } = require('../../fixtures');
+
+const { BANK1_MAKER1, ADMIN } = MOCK_USERS;
+
+const filters = dashboardFilters;
+const statusCheckboxSelectors = dashboardDeals.filters.panel.form.status;
+
+context('Dashboard Deals Cancellation status filter - Deal cancellation enabled', () => {
```

Comment:
Can we say "- X feature flag enabled" / "disabled" throughout please just so it's clear